package.json

```diff
@@ -67,7 +67,7 @@
     "@sinonjs/commons": "^1.7.0",
     "@sinonjs/fake-timers": "^6.0.0",
     "@sinonjs/text-encoding": "^0.7.1",
-    "just-extend": "^4.0.2",
+    "object-assign": "^4.0.0",
```

Comment:
The `object-assign` polyfill is slightly _bigger_ than the `just-extend` polyfill

```
$ ls -la node_modules/object-assign/index.js
-rw-r--r--  1 morgan  staff  2108 Jan 16  2017 node_modules/object-assign/index.js
$ ls -la node_modules/just-extend/index.js
-rw-r--r--  1 morgan  staff  1947 Oct 26  1985 node_modules/just-extend/index.js
```

In the file system I'm using, that translates to 4k per file

```
$ du -h node_modules/object-assign/index.js
4.0K	node_modules/object-assign/index.js
$ du -h node_modules/just-extend/index.js
4.0K	node_modules/just-extend/index.js
```